plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -175,9 +175,9 @@ private BlockBasedTableConfig createBlockBasedTableConfigHighSpec() {
     final LRUCache cache = new LRUCache(ROCKSDB_BLOCKCACHE_SIZE_HIGH_SPEC);
     return new BlockBasedTableConfig()
         .setBlockCache(cache)
-        .setFormatVersion(ROCKSDB_FORMAT_VERSION)
-        .setOptimizeFiltersForMemory(true)
-        .setCacheIndexAndFilterBlocks(true)
```

Comment:
are these other changes required for bloom filters or are they uncorrelated?

Reply:
Yes, these are related to bloom filters,   setCacheIndexAndFilterBlocks is by default set to false to avoid putting in cache indexes and filters, this will have an impact on cache hit ratio for other cached data (real data). I did tests with setOptimizeFiltersForMemory(true) and without and didn't see any difference in terms of memory consumption, as I wanted to keep the configuration simple, I delete it.